src/2_RTB/NormeDiProgetto/changelog.csv

```diff
@@ -3,4 +3,5 @@
 0.3,13-11-2023,Aggiunte sezioni "Oggetti Esterni al Repository" e "Tracciamento del Tempo Speso"; modificata organizzazione logica,Matteo Rango,Amministratore
 0.4,14-11-2023,Aggiunta sezione "Processi Primari"; Chiarito workflow documentazione e codice,Matteo Rango,Amministratore
 0.5,17-11-2023,Aggiunta specifica inserimento termini nel Glossario,Matteo Rango,Amministratore
-0.6,24-11-2023,Aggiunta sezione "Analisi requisiti",Riccardo Costantin, Amministratore
\ No newline at end of file
+0.6,24-11-2023,Aggiunta sezione "Analisi requisiti",Riccardo Costantin, Amministratore
+0.7,24-11-2023,Aggiunta sezione "Ruoli progetto",Riccardo Costantin,Amministratore
```

Comment:
Tecnicamente, dovremmo aggiungere una nuova entry al changelg ogni volta che modifichiamo il file, soprattutto in seguito ad una revisione; questo significa che dovrebbe esserci un entry 0.7 per le modifiche fatte ieri alla sezione Analisi dei Requisiti, una 0.8 per l'aggiunta della sezione Ruoli progetto e una 0.9 non appena modificherai quest'ultima sezione.

Reply:
dalle norme di progetto

Due modiﬁche, fatte in momenti diversi, differiscono l’una dall’altra solo se
hanno scopi diversi. Ad esempio non è necessario incrementare la versione se
viene fatta una modiﬁca alla stessa sezione in due giorni differenti; anche se
faccio una modiﬁca, ed essa non viene approvata, non è necessario
incrementare la versione con le nuove modiﬁche proposte dal/dai
veriﬁcatore/i, dal momento che modiﬁca e veriﬁca “viaggiano”
parallelamente.

Reply:
touché